easybuild/tools/run.py

```diff
@@ -155,6 +155,13 @@ def run_cmd(cmd, log_ok=True, log_all=False, simple=False, inp=None, regexp=True
     else:
         raise EasyBuildError("Unknown command type ('%s'): %s", type(cmd), cmd)
 
+    if shell is None:
+        shell = True
+        if isinstance(cmd, list):
+            raise EasyBuildError("When passing cmd as a list then `shell` must be set explictely! "
+                                 "Not that all elements of the list but the first are treated as arguments "
```

Comment:
s/Not/Note/

Reply:
Done.
PS: Check the "+-" button to directly make a suggestion of a code change.